src/sentry/integrations/slack/requests.py

```diff
@@ -92,7 +96,22 @@ def _validate_data(self):
             raise SlackRequestError(status=400)
 
     def _authorize(self):
-        if self.data.get("token") != options.get("slack.verification-token"):
+        signing_secret = options.get("slack.signing-secret")
```

Comment:
can you leave a `TODO` here so that we know we'll need to update this to also check `slack.v2.signing-secret` (or whatever we call it) whenever that gets added in?